crates/context/README.md

```diff
@@ -0,0 +1,65 @@
+# Calimero Contexts
+
+- [Context Lifecycle](#context-lifecycle)
+- [Administrative Operations](#administrative-operations)
+- [Context State](#context-state)
+- [State Mutations](#state-mutations)
+- [Context Membership](#context-membership)
+  - [Invitations](#invitations)
+  - [Leaving](#leaving)
+- [Encryption](#encryption)
+- [Application Upgrade](#application-upgrade)
+- [Context Deletion](#context-deletion)
+
+A context in Calimero is an instance of a deployed application members of which share a synchronized state.
+
+A context is identified by a 32-byte key, which is randomly generated when the context is created. As of now, the key is not necessarily cryptographically secure, nor is it a dependency of cryptographic operations, but it is unique enough to mitigate collisions. Worth noting that this is likely to change in the future.
+
+All operations within a context performed by members will be broadcasted to all other members. This includes:
+
+- State mutations
+- Membership changes (invitation, leaving, kickouts)
+
+## Context Lifecycle
+
+Following the creation of a context, the only member is the creator. The creator can invite other members to join the context, and members can leave the context at any time.
+
+## Administrative Operations
+
+Application authors can define gated behaviour for administrative operations, but by default all members have the same rights. Which, depending on the application, can be an undesirable behaviour.
```

Comment:
"Application authors can define gated behaviour for administrative operations" <- do we really have this atm?

Reply:
> do we really have this atm?

depends on how you define this

do we have applications that do this right now? no

can app developers do this if they wanted to? yep

Reply:
removed